esmf_regrid/tests/unit/experimental/unstructured_scheme/test__cube_to_GridInfo.py

```diff
@@ -0,0 +1,94 @@
+"""Unit tests for miscellaneous helper functions in `esmf_regrid.experimental.unstructured_scheme`."""
+
+from iris.coords import DimCoord
+from iris.cube import Cube
+import numpy as np
+import scipy.sparse
+
+from esmf_regrid.esmf_regridder import Regridder
+from esmf_regrid.experimental.unstructured_scheme import _cube_to_GridInfo
+
+
+def _grid_cube(n_lons, n_lats, lon_bounds, lat_bounds, circular=False):
+    lon_lower, lon_upper = lon_bounds
```

Comment:
Lines 13-17 are identical to 22-27.  And only the `ll_bound_span` needs to be returned in both cases.
Lets refactor and tidy this up - then its looking good.